src/libraries/Microsoft.Extensions.Http/src/DefaultHttpClientFactory.cs

```diff
@@ -194,13 +194,13 @@ internal void ExpiryTimer_Tick(object? state)
         }
 
         // Internal so it can be overridden in tests
```

Comment:
Does the unit test project have a friend relationship with the implementation project? This comment certainly implies sealing the type / methods would be a break.